src/pool/srv_pool.c

```diff
@@ -3015,17 +3015,19 @@ ds_pool_svc_get_prop(uuid_t pool_uuid, d_rank_list_t *ranks,
 }
 
 int
-ds_pool_reintegrate(uuid_t pool_uuid, d_rank_list_t *ranks,
-		uint32_t reint_rank, struct pool_target_id_list *reint_list)
+ds_pool_target_update(uuid_t pool_uuid, d_rank_list_t *ranks,
```

Comment:
I wonder if it's worth naming this `ds_pool_target_update_state()` to be more specific, as that's what's really being updated right?